cdap-master/pom.xml

```diff
@@ -58,12 +58,6 @@
     </dependency>
     <dependency>
       <groupId>co.cask.cdap</groupId>
-      <artifactId>cdap-common</artifactId>
-      <version>${project.version}</version>
-      <type>test-jar</type>
-    </dependency>
```

Comment:
did we run master on cluster with this change, this shouldn't hopefully break starting master.